src/utils/geolocationUtil.jsx

```diff
@@ -14,8 +14,7 @@ function handleError(error) {
       break;
   }
 }
-
-export function useCurrentLocation() {
+export function geolocationUtil() {
```

Comment:
pls rename it in accordance with the purpose of this function

Reply:
done